daemon/cmd/daemon_main.go

```diff
@@ -1454,6 +1458,15 @@ func initEnv(cmd *cobra.Command) {
 			)
 		}
 	}
+
+	// NetfilterCompatibleMode should be enabled only on higher versions (5.4 or later)
+	// of kernel. If kernel doesn't support higher code memory size, then disable NetfilterCompatibleMode.
+	if option.Config.NetfilterCompatibleMode {
+		if !probes.NewProbeManager().GetMisc().HaveLargeInsnLimit {
+			option.Config.NetfilterCompatibleMode = false
+			log.Warnf("NetfilterCompatibleMode is enabled only on 5.4 and higher versions of kernel.")
```

Comment:
Please use the flag name in the warning instead of `NetfilterCompatibleMode`

Reply:
As suggested, will fix in separate PR.